include/RAJA/exec-cuda/scan_cuda.hxx

```diff
@@ -80,8 +80,7 @@ template <typename InputIter, typename Function, typename T>
 void inclusive_inplace(const ::RAJA::cuda_exec_base&,
```

Comment:
There are leftover template parameters here that actually break the cuda-enabled build. I'm fixing them in #139